OpenRA.Mods.Common/Lint/CheckSequences.cs

```diff
@@ -89,6 +90,48 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Map map)
 						}
 					}
 				}
+
+				foreach (var weaponInfo in map.Rules.Weapons)
+				{
+					var projectileInfo = weaponInfo.Value.Projectile;
+					if (projectileInfo == null)
+						return;
```

Comment:
The very first weapon definition in TD triggers this case, so the pass is not run at all.